lib/Entrypoint.js

```diff
@@ -41,6 +41,9 @@ Entrypoint.prototype.getSize = function(compilation) {
 	var files = this.getFiles();
 
 	return files
+		.filter(function(asset) {
+			return !(/\.map$/.test(asset))
+		})
```

Comment:
Maybe it makes sense to remove the Entrypoint.prototype.getSize method altogether and do it somewhere else (in the plugin). It has a weird Compilation dependency and now also a SourceMap dependency...